browser/net/brave_proxying_url_loader_factory.cc

```diff
@@ -26,66 +27,6 @@
 
 namespace {
 
-// User data key for ResourceContextData.
-const void* const kResourceContextUserDataKey = &kResourceContextUserDataKey;
-
-class ResourceContextData : public base::SupportsUserData::Data {
```

Comment:
These two classes are common for `BraveProxyingURLLoaderFactory` and `BraveProxyingWebSocket`, so they are extracted to a separate *.h/*.cc